README.md

```diff
@@ -19,8 +19,10 @@ For Decanter v7, instead of using SASS/SCSS with the BEM naming convention, we u
 We have removed all assets from the repo in Decanter v7. Instead, we are using remote third party resources for our fonts and icons.
 
 ### Fonts
-- For Source Sans Pro, Source Serif Pro, Roboto Slab, Roboto Mono - we include them using the `@import` method from [Google Fonts](https://fonts.google.com/).
-- For the Stanford ligature font that we use for the logo, we link them from the [University Communications media CDN](https://www-media.stanford.edu/assets/fonts/stanford.woff).
+- We do not include any font assets with Decanter v7. If you want to use the Decanter fonts in your own projects, it is recommended that you use the font loading method that is optimized for your framework and only import the fonts that you need.
```

Comment:
We can also encourage our clients to import our fonts.css file in their CSS file as we do in alumni. 

https://github.com/SU-SWS/saa_alumni/blob/dev/src/styles/global.css#L1

This way, when we change, they change.

Reply:
This makes sense - I might update this to split the import into 2 files - one for essential sans and serif, the other also include the slab and monospace fonts 🤔